### ISSUE TITLE
Test with MySQL 5.7

### ISSUE THREAD

**Issue by rbock** [rbock/sqlpp11-connector-mysql#3]
I received an offline report saying that some of the current tests fail due to bad table layout.

**Comment by AndiDog** [comment]
While you're at it: it would also be great to have e.g. "TabSample.h" autogenerated because the header differs from the SQL file (example: gamma column of type bool vs. varchar). Makes later changes easier.

**Comment by digitalist** [comment]
```
Your MySQL connection id is 19
Server version: 5.7.12-0ubuntu1 (Ubuntu)
```

all tests passed with current master/connector (except move constructor, but it's not related to server version)

**Comment by rbock** [comment]
Thanks for the hint.

Actually, I wonder why some tests worked fine on Linux, although the tests themselves were buggy.

MySQL on Mac seems much less forgiving. Fixed a few things with the new release 0.16.
